frontend/src/plaquettes/Template.js

```diff
@@ -263,13 +128,9 @@ export default class Template {
     }
     // Render the plaquette
     const plaquette = new Plaquette(this.selectedQubits, this.workspace, this.app);
-    // Remove seleected qubits from the template qubits, so they can be
```

Comment:
To be clear, I don't think a `Footprint` should remember the specific set of qubits (in the viewport) used to create it, but should remember a cell of qubit positions (just like the unit cell) for re-creating the footprint anywhere on the viewport. Can we add that to this class?

Reply:
yes, would you mind pointing me to how we are instantiating the qubit positions?

Reply:
When it is placed, that information should be known to the workspace. When it is stored (say, in redux), we only need to store the *relative* positions. There's a 1-1 correspondence, and storing relative positions helps us keep things general.

Reply:
Okay makes sense, w.r.t. relative position could you point me towards any existing code that lays out the foundation?